util-taglib/src/com/liferay/taglib/aui/ScriptTag.java

```diff
@@ -139,6 +142,38 @@ public int doEndTag() throws JspException {
 				bodyContentSB = sb;
 			}
 
+			if (load != null) {
```

Comment:
This is copied from `JSModuleLauncherImpl` because we have no way to access the service from this module (which is outside of OSGi).

However, because this code is almost a spec, there's not so much risk. Also, `ScriptTag` remains for temporary compatibility, but it should die slowly :crossed_fingers: .